specificationRepositoryConfiguration.json

```diff
@@ -8,8 +8,8 @@
       "configFilePath": "eng/swagger_to_sdk_config.json"
     },
     "azure-sdk-for-java": {
-      "integrationRepository": "azure-sdk/azure-sdk-for-java",
-      "mainRepository": "Azure/azure-sdk-for-java",
+      "integrationRepository": "test-repo-billy/azure-sdk-for-java",
+      "mainRepository": "test-repo-billy/azure-sdk-for-java",
```

Comment:
Add `mainbranch` config to point to your personal branch if your change is on your personal branch rather than in main branch.